OpenSim/Actuators/Thelen2003Muscle.h

```diff
@@ -205,16 +184,9 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Thelen2003Muscle, ActivationFiberLengthMuscle);
     */
     double getMaximumPennationAngle() const;
 
-    bool setActivationTimeConstant(double aActivationTimeConstant);   
-    bool setDeactivationTimeConstant(double aDeactivationTimeConstant);
-    bool setMinimumActivation(double aActivationMinValue);
-    bool setFmaxTendonStrain(double aFmaxTendonStrain);
-    bool setFmaxFiberStrain(double aFmaxMuscleStrain);
-    bool setKshapeActive(double aKShapeActive);
-    bool setKshapePassive(double aKshapePassive);
-    bool setAf(double aAf);
-    bool setFlen(double aFlen);
-    bool setForceVelocityExtrapolationThreshold(double aFvThresh);
+    void setActivationTimeConstant(double actTimeConstant);   
+    void setDeactivationTimeConstant(double deactTimeConstant);
+    void setMinimumActivation(double minimumActivation);
```

Comment:
Otherwise users might wonder "which way am I supposed to edit the properties? are they different?" One camp might say to only allow one way to do something, but I think the convenience is nice.